tests/cpydiff/core_class_name_mangling.py

```diff
@@ -0,0 +1,25 @@
+"""
+categories: Core,Classes
+description: Private Class Members name mangling is not implemented
+cause: THe MicroPython compiler has not yet implemented name mangling for private class members at runtime.
```

Comment:
Typo and saying "not yet" makes it sound like there are plans to implement it, which is not currently true.

```suggestion
cause: The MicroPython compiler does not implement name mangling for private class members at runtime.
```

Reply:
Git blame says the comment is from the initial commit of MicroPython 10 years ago, so likely doesn't reflect the current state of things. I guess Damien can chime in here if he wants, but it seems like if it hasn't been implemented by now, we don't really need it.

Reply:
The vast majority of tests in `tests/cpydiff` use language like "not implemented", so I suggest to also use that language here, ie remove the "yet".

Then I think it's a good idea to remote that TODO in `compile.c`.  This test will encode the intention of that to-do.

Reply:
I updated the wording as suggested, and removed the TODO in `compile.c`.